doc/source/examples/extended_examples/gui/executable.rst

```diff
@@ -0,0 +1,118 @@
+.. _gui_example:
+
+=======================================
+Create a GUI app in Python with PySide6
+=======================================
+
+This example shows how to create a graphical user interface (GUI) app in Python that uses PyMAPDL to compute the deflection of a square beam.
+
+Simulation setup
+================
+
+The :download:`gui.py <gui.py>` script launches a graphical app using PySide6.
+The preprocessing tab contains input fields for Poisson's ratio, Young modulus, beam length, and number of simulation points.
+
+.. literalinclude:: gui.py
+
+.. image:: base_app.png
+
+Add a PyVista plotting frame in the window
+==========================================
+
+Start by importing ``QtInteractor` from ``pyvistaqt`` and ``MapdlTheme`` from ``ansys-mapdl-core``.
+
+.. code:: python
+
+    from pyvistaqt import QtInteractor
+    from ansys.mapdl.core import MapdlTheme
+
+Then, add a plotter in the first tab:
+
+.. code:: python
+
+    def _setup_tab_preprocessing(self) -> None:
+        ...
+        container_layout.addWidget(self._run_preprocessor_button, 5, 0, 1, 3)
+
+        # PyVista frame in the window
+        self._preprocessing_plotter = QtInteractor(theme=MapdlTheme())
+        container_layout.addWidget(self._preprocessing_plotter, 0, 4, 6, 50)
+
+And another one in the second tab:
+
+.. code:: python
+
+    def _setup_tab_postprocessing(self) -> None:
+        container_layout = QtWidgets.QVBoxLayout()
+        self._tab_postprocessing.setLayout(container_layout)
+        self._postprocessing_plotter = QtInteractor(theme=MapdlTheme())
+        container_layout.addWidget(self._postprocessing_plotter)
+        self._deflection_label = QLabel("Deflection: ")
+        container_layout.addWidget(self._deflection_label)
+
+Finally, make sure to correctly close the widget using the app:
+
+.. code:: python
+
+    def closeEvent(self, event) -> None:
+        self._preprocessing_plotter.close()
+        self._postprocessing_plotter.close()
+        event.accept()  # let the window close
+
+Import MAPDL instance in your window
+====================================
+
+Add an attribute to your MainWindow for the MAPDL instance and import ``launch_mapdl``.
+
+.. code:: python
+
+    from ansys.mapdl.core import Mapdl, launch_mapdl
+
+.. literalinclude:: gui_app.py
+    :lines: 22-26
+
+.. literalinclude:: gui_app.py
+    :lines: 231-236
```

Comment:
I think this should work.
```suggestion
.. literalinclude:: gui_app.py
    :lines: 19, 22-26, 231-236
```

Reply:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude